src/backend/render/mod.rs

```diff
@@ -913,7 +914,12 @@ where
     }
 
     renderer.bind(target).map_err(RenderError::Rendering)?;
-    let res = damage_tracker.render_output(renderer, age, &elements, CLEAR_COLOR);
+    let res = damage_tracker.render_output(
+        renderer,
+        age,
+        &elements,
+        crate::theme::clear_color(state.theme.cosmic()),
```

Comment:
Testing an implementation of ext-session-lock, I  noticed if I passed a different color to `damage_tracker.render_output` when the session lock was enabled, it ended up flickering between that and the previous color. The damage tracker I guess doesn't automatically recognize a change to the clear color as damage.

So I think  to change `clear_color` dynamically without possible damage tracking issues, Smithay or cosmic-comp may need some change here to mark everything as damaged when this changes?

Reply:
ok, I can make the clear color static for now.